app/gui/src/dashboard/components/AriaComponents/Button/Button.tsx

```diff
@@ -337,9 +337,7 @@ const ButtonContent = memo(function ButtonContent(props: ButtonContentProps) {
         styles={styles}
         hideLoader={hideLoader}
       />
-      <Text color="inherit" truncate="1" className={styles.text()}>
```

Comment:
We used to have a span with `className={styles.text()}`, haven't we?

Reply:
well, i tried extracting the css directly and it doesn't seem to work.
try running this commit in dev (staging env) and create a project execution (no repeats to minimize load, but long timezone to (try to) trigger the `truncate` class):

https://github.com/enso-org/enso/pull/12397/commits/438c4f3120ae91304b5788611366a03d12f524fe